ambry-api/src/main/java/com.github.ambry/config/StoreConfig.java

```diff
@@ -216,6 +216,14 @@
   public static final String storeReadWriteEnableSizeThresholdPercentageDeltaName =
       "store.read.write.enable.size.threshold.percentage.delta";
 
+  /**
+   * Don't do zero copy but pre-fetch data from disk if true.
+   */
+  @Config(storeEnableDataPrefetchName)
```

Comment:
wondering if this should be a `ServerConfig` not a `StoreConfig` since it is not store code that initiates the prefetch